x-pack/plugins/security/index.js

```diff
@@ -116,50 +114,48 @@ export const security = (kibana) => new kibana.Plugin({
     server.auth.strategy('session', 'login', 'required');
 
     const auditLogger = new SecurityAuditLogger(server.config(), new AuditLogger(server, 'security'));
-    if (config.get('xpack.security.rbac.enabled')) {
```

Comment:
Apologies for the huge diff, this was just removing this ^^ if check and the indentation change makes it look larger than it is.